int32 localization_request
---
int32[1] localization
